www_src/pages/section-2/page.jsx

```diff
@@ -0,0 +1,19 @@
+var React = require('react');
+
+module.exports = React.createClass({
+  render: function () {
+    var style = {};
+
+    if (this.props.screenshot !== 'EMPTY') {
```

Comment:
Since this was just a temporary thing anyway I'm going to use an empty object instead, which will eventually become a *meaningful* page object.